modules/api/docs/LorisRESTAPI_v0.0.4-dev.md

```diff
@@ -823,90 +823,191 @@ The JSON object is of the form:
 
 ## 5.0 DICOM Data
 
-Like the imaging data, the DICOM data mostly lives in the `/candidates/$CandID/$Visit` 
-portion of the REST API namespace, but is defined in a separate section of this 
+Like the imaging data, the DICOM data mostly lives in the `/candidates/$CandID/$Visit`
+portion of the REST API namespace, but is defined in a separate section of this
 document for clarity purposes.
 
 ### 5.1 Candidate DICOMs
+
 ```
-GET /candidates/$CandID/$Visit/dicoms
+GET /candidates/$CandID/$VisitLabel/dicoms
 ```
 
 A GET request to `/candidates/$CandID/$Visit/dicoms` will return a JSON object of
-all the raw DICOM data which have been acquired for that visit. It will return an 
+all the DICOM archives which have been acquired for that visit. It will return an
 object of the form:
 
 ```js
 {
     "Meta" : {
         "CandID" : $CandID,
-        "Visit" : $VisitLabel,
+        "Visit" : $Visit,
     },
-    "DicomTars" : 
-    [
+    "DicomArchives" : [
         {
-            "Tarname" : "DCM_yyyy-mm-dd_ImagingUpload-hh-mm-abc123.tar",
-            "Patientname" : "DCM123_123456_V1",
-            "SeriesInfo" :
-                [{
+            "ArchiveName" : "DCM_yyyy-mm-dd_ImagingUpload-hh-mm-abc123.tar",
+            "PatientName" : "DCM123_123456_V1",
+            "SeriesInfo" : [
+                {
                     "SeriesDescription" : "MPRAGE_ipat2",
-                    "SeriesNumber" : "2",
-                    "EchoTime" : "2.98",
-                    "RepetitionTime" : "2300",
-                    "InversionTime" : "900",
-                    "SliceThickness" : "1",
+                    "SeriesNumber" : 2,
+                    "EchoTime" : 2.98,
+                    "RepetitionTime" : 2300,
+                    "InversionTime" : 900,
+                    "SliceThickness" : 1,
                     "Modality" : "MR",
                     "SeriesUID" : "1.2.3.4.1107",
-                    },
-                    {
+                },
+                {
                     "SeriesDescription" : "BOLD Resting State",
-                    "SeriesNumber" : "5",
-                    "EchoTime" : "30",
-                    "RepetitionTime" : "2100",
+                    "SeriesNumber" : 5,
```

Comment:
If they are not strings it's a bug and the code and should be fixed in a different PR. Numbers are not reliable in JSON since javascript only has a floating point data type to represent numbers.

Reply:
I think using numbers to represent floats and ints in the code is not a bug and but rather a good choice here, as we already use these representations in the database:
- For floating numbers, we go from doubles in the database to doubles in the code, so no loss of precision here.
- For integers, the values we represent are well below `Number.MAX_SAFE_INTEGER` (9007199254740991), so there should not be any problem as well.

Reply:
While it may be debateable whether these could have originally been safely represented as numbers, they were encoded as strings because strings faithfully represent the value given to them, while numbers can differ for various reasons related to the fact that javascript only includes floating point number (ie. values could be not representable, trailing 0s that might be significant for fixed point numbers or versions, etc) and this part of the API is already released so should not be changed.

Reply:
Hhhhm, this makes sense I'll change it back. Since I was not the one who originally wrote the PR I was not sure what was already part of the PR when I touched it :thinking:.